examples/simulation-pytorch/sim.py

```diff
@@ -211,6 +203,7 @@ def main():
         config=fl.server.ServerConfig(num_rounds=args.num_rounds),
         strategy=strategy,
         actor_kwargs={
+            # TODO: I used to need this but, is it still needed?
```

Comment:
I think it'll be needed because I have not disabled the progress bar in the library by default. (yet it should return fewer things since the `map` was creating a progress bar for each transform, which is no longer used)